docs/docker_container.md

```diff
@@ -48,19 +48,22 @@ pip3 install ovmsclient
 ```
 
 Run prediction
-```python
-import numpy as np
+```bash
+echo 'import numpy as np
```

Comment:
Here we loose the syntax highlighting for python script

Reply:
To use this python code in test I need it in form of an executable command. This is why I saved in in file which then I can use. To preserve syntax highlighting I could add it for the second time with python keyword? But then every change in code would have to be made twice.

Reply:
I think its not worth duplicating the snippet.

OVMS will be receiving new docs and updates to existing docs in the future. It looks like from now on, devs need to write the doc with automated validation tests in mind. I think we should be instructed how to do so, otherwise we will end up with bugs by the end of each release. @ngrozae

Reply:
Instructions added to jira ticket for now https://jira.devtools.intel.com/browse/CVS-85355. I will move it to ovms wiki when I get access

Reply:
https://wiki.ith.intel.com/display/OVMS/Writing+documentation+for+automated+tests